Source/santactl/Commands/SNTCommandRule.m

```diff
@@ -181,6 +189,16 @@ - (void)runWithArguments:(NSArray *)arguments {
         [self printErrorUsageAndExit:@"--import requires an argument"];
       }
       jsonFilePath = arguments[i];
+    } else if ([arg caseInsensitiveCompare:@"--clean"] == NSOrderedSame) {
+      if (!importRules) {
```

Comment:
It might be nice to consider moving this check below this chunk of code checking arguments. As written, you cannot do `santactl rule --clean --import ...` which might be confusing.

Maybe something below like:

```
if (!importRules && cleanupType != SNTRuleCleanupNone) {
    // [self printErrorUsageAndExit...]
}
```

If order does matter for some reason and you'd like to keep as is, might be worth documenting the requirement in the usage string.